backend/main.py

```
@@ -66,7 +66,7 @@ def read_careers():
             columns = [desc[0] for desc in cursor.description]
             return [dict(zip(columns, row)) for row in rows]
     except Exception as e:
-        raise HTTPException(status_code=500, detail=str(e))
+        raise HTTPException(status_code=500, detail="Internal Server Error")
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)